rpc/client.go

```diff
@@ -129,9 +129,9 @@ func (c *Client) connect(opts *retry.Options, context *Context) {
 
 	err := retry.WithBackoff(retryOpts, func() (retry.Status, error) {
 		conn, err := tlsDialHTTP(c.addr.Network(), c.addr.String(), context.tlsConfig)
-		if err != nil {
-			log.Info(err)
-			return retry.Continue, nil
+		if err != nil { //could be many error: bad host; bad certificate
```

Comment:
Let's put this comment on a separate line. Also, add a space between `//` and `could`, capitalize `Could` and add a period to match with our commenting style.